### ISSUE TITLE
AssertionError: Errors in swagger.yaml:

### ISSUE THREAD

**Issue by johnneerdael** [Parquery/swagger-to#74]
Hi,

I'm running in to some issues converting a yaml swagger page, at first I noticed some issues in the yaml page on editor.swagger.io but I've resolved those, now I'm trying to resolve the current issue but the error is a bit vague to find out where the issue in my file is exactly? Link to my YAML file: https://pastebin.com/W2sLJxEF

```
➜  Scripts swagger_to_py_client.py --swagger_path swagger.yaml --outpath ipamsvc_module.py --force
Traceback (most recent call last):
  File "/usr/local/bin/swagger_to_py_client.py", line 57, in <module>
    main()
  File "/usr/local/bin/swagger_to_py_client.py", line 34, in main
    raise AssertionError("Errors in {}:\n{}".format(swagger_path, "\n".join(errs)))
AssertionError: Errors in swagger.yaml:
missing tag "name" in the swagger specification
```

**Comment by mristin** [comment]
Hi @Johnneke ,
You need to specify the name of the service as a tag. This is required so that the generated code can provide informative documentation (and also to designate the package names, *e.g.*,  in the go server code).

Have a look at the Uber example:
https://github.com/Parquery/swagger-to/blob/7497578940cb03ec4c4cdb1f21c48c19177b709b/tests/cases/py_client/general/swagger.yaml#L10

I'll update the readme.

**Comment by mristin** [comment]
Here is the updated section in the Readme: https://github.com/Parquery/swagger-to/blob/master/README.rst#usage

Please note that your Swagger spec revealed another bug (parameter names starting with '_' such as '_field'). I am working on the patch.

**Comment by mristin** [comment]
I've just published the patch version 3.1.3. There are no exceptions -- could you please confirm that the generated code behaves as expected?

**Comment by mristin** [comment]
Closing since the issue seems resolved.
